README.md

```diff
@@ -22,7 +22,85 @@
 
 # 1. Basic Operations
 
-### a. `export`
+### a. `type`
+Displays the kind of command the shell will execute, given a particular command name. 
+```bash
+type command
+```
+Example:
+```bash
+$ type cp
+cp is /bin/cp
+```
+### b. `help`
+It is used to get information about each of the shell builtin commands.
+```bash
+help command
+```
+Optionally, the -m option is added to change the format of the output.
+Example:
+```bash
+$ help -m cd
```

Comment:
Remove the details of cd command. Only what help command is and how to use it and a example is ok. All below lines create confusion.